exporter/collector/integrationtest/metric_test_server.go

```diff
@@ -150,7 +159,6 @@ func (m *MetricsTestServer) NewExporter(
 ) *collector.MetricsExporter {
 	cfg.MetricConfig.ClientConfig.Endpoint = m.Endpoint
 	cfg.MetricConfig.ClientConfig.UseInsecure = true
-	cfg.ProjectID = "fakeprojectid"
```

Comment:
removed this because it was redundant with the setting in [`testcase.CreateMetricConfig()`](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/5ca7a23/exporter/collector/integrationtest/testcase.go#L360) and overwrote any attempt to set `ProjectID` with a `Configure()` func

Reply:
Hmmm... was this required?  The test doesn't pass anymore

Reply:
weird, it passes for me locally.

the failure also looks like it's just the entries aren't in the expected order: https://app.circleci.com/pipelines/github/GoogleCloudPlatform/opentelemetry-operations-go/1387/workflows/a7049ae8-6a76-4a78-9baa-a842d951eaee/jobs/1382?invite=true#step-105-585

could that just be the case?

Reply:
ahh... that makes sense

Reply:
thanks